Sources/Core/Components/PlayerPlugin.swift

```diff
@@ -27,17 +27,38 @@
 import AVFoundation
 
 public protocol PlayerPlugin {
-    /**
-    Method called when Player is in Init state
-     - parameters:
-        - player: instance of AVPlayer used
-     */
+    ///
+    /// Called when Player init Initialization state
+    /// - parameters:
+    ///    - player: instance of AVPlayer used
+    ///
     func didInit(player: AVPlayer)
     
-    /**
-    Method called when Player is in Loaded state
-     - parameters:
-        - media: PlayerMedia just loaded
-     */
-    func didLoadMedia(_ media: PlayerMedia?)
+    /// Called when Player init Loading state
+    func startLoading()
```

Comment:
You should keep methods name consistency: 
- didStartLoading
- didStartBuffering
...

Reply:
right!